packages/insomnia-app/app/common/har.js

```diff
@@ -489,6 +490,13 @@ function getRequestPostData(renderedRequest: RenderedRequest): HarPostData | voi
   let params = [];
   if (body.params) {
     params = body.params.map(param => {
+      if (param.type === 'file') {
+        return {
+          name: param.name,
+          value: param.fileName,
```

Comment:
While this now correctly includes the path in the generated curl command, it doesn't actually work correctly when you run it, as curl needs a file path to be prepended with `@` in order for it to read it.

If we set `fileName` instead of `value`, httpsnippet will do so:
https://github.com/Kong/httpsnippet/blob/dc08d8338a5e27caa3bdddb0131c3ed8eb92080c/src/targets/shell/curl.js#L48-L52

```suggestion
          fileName: param.fileName,
```

Reply:
Arf good catch I should have took a look at httpsnippet 🤦